.vsts-pipelines/templates/phases/default-build.yml

```diff
@@ -10,12 +10,15 @@
 #       Additional steps to run before build.sh/cmd
 #   afterBuild: [steps]
 #       Additional steps to run after build.sh/cmd
+#   artifactsPath: string
+#       The path to the folder of build output to capture. Defaults to artifacts/build/
```

Comment:
Worth mentioning that setting to empty is how to disable publish (assuming I am reading this PR properly that is)?